model/input_pipeline.py

```diff
@@ -285,18 +285,20 @@ def filter_fn(x):
   dataset = dataset.repeat(num_epochs)
 
   if pack_examples:
-    dataset = pack_dataset(dataset, max_length)
+    dataset = pack_dataset(dataset, max_length, keys = ["inputs", "targets"])
```

Comment:
Nit: single quotes to be consistent in this file.
```suggestion
    dataset = pack_dataset(dataset, max_length, keys = ['inputs', 'targets'])
```